cfpb/cfgov-refresh
# Removing unused event landing page file

Removing unused event file

## Removals

- Removed `cfgov/jinja2/v1/event_landing_page.html` file, which isn't being used.

## Testing

- Search for references to `cfgov/jinja2/v1/event_landing_page.html` in the repo.
- Visit `http://localhost:8000/about-us/events/` and verify that it works.
- Visit `http://localhost:8000/about-us/events/request-speaker/` and verify that it works.
- Visit `http://localhost:8000/about-us/events/archive-past-events/` and verify that it works.


## Checklist
